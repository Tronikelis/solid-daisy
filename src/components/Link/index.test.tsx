import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Link } from ".";

describe("<Link />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Link>huh</Link>);
        expect(container).toMatchSnapshot();
    });
});
