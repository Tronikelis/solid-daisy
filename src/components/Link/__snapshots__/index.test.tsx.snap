// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Link /> > renders correctly 1`] = `
<div>
  <a
    class="link"
  >
    huh
  </a>
</div>
`;
